### ISSUE TITLE
PLEASE remove dist folder from .gitignore

### ISSUE THREAD

**Issue by mithray** [textileio/js-ipfs-lite#399]
Please, just simply remove the dist line from .gitignore, it would be SOOOO helpful so that you git repo will include a prebuilt file that can be used in the browser.

When trying to build this, I am forced to download Chromium(ugh, WHY?!?!, why download 120mb just to build this?), and webpack throws errors at me about peer dependencies I need to manually install. In addition I have multiple versions of webpack installed and none of them are detected. I could work this out, but it's a pain to do this with many dependencies!

In addition to this, maybe it does compile for me finally, what's to say a space is missing, I get a different hash, and then the file isn't massively available from IPFS peers? Doesn't it make so much more sense on a content addressed network to have widely shared official, canonical versions  ?

**Comment by mithray** [comment]
Also, i'm curious if this library isn't intentionally obscure to use for some political reason, as the README on npmjs shows as empty. As far as I know, the only way to achieve this is by committing a README to github, but then deleting before every npm publish. Otherwise why would npmjs not show the readme? Apparently there have been 191 commits, so it's not because of some initial misconfiguration that was forgotten. 



https://www.npmjs.com/package/dag-service

**Comment by carsonfarmer** [comment]
that might just be because this library was migrated over from the @textileio org :) Not intentionally leaving out the README. But for npm to have the README, you need to have it in the deploy folder. Perhaps we aren't pushing that in there. That's also the reason we don't have our dist folder in version control... it gets generated in CI on deploy. But both of these things can be changed :)

**Comment by jacobheun** [comment]
fixed by https://github.com/ipfs/js-dag-service/pull/400

**Comment by mithray** [comment]
Thank you all kindly. Sorry if I seemed antagonistic at all. I did honestly wonder if there might be some political or LICENSE reason that a README would be deployed to Github but not npmjs as it actually seemed like considerably _more_ work to separate the two. I guess whatever build steps you are using make this easier than usual.

If this library is really functional, I don't know why it would not find some popularity on NPM. The plurality of IPFS libraries of course is somewhat confusing, but having a NPM version number 0.0.0-dev and no readme certainly adds to its obscurity.
